### PR TITLE
Enable central package versions management

### PR DESCRIPTION
**Pull request by kevinchalet**
https://github.com/NuGet/Home/wiki/Centrally-managing-NuGet-package-versions

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<Project>
+
+  <ItemGroup>
+    <PackageVersion Include="AngleSharp" Version="0.13.0" />
+    <PackageVersion Include="JetBrains.Annotations" Version="2020.1.0" />
+    <PackageVersion Include="JustEat.HttpClientInterception" Version="3.0.0" />
+    <PackageVersion Include="MartinCostello.Logging.XUnit" Version="0.1.0" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="3.1.3" />
```
Comment (martincostello) [inline Directory.Packages.props]: The 3.1.3 and packages used by the sample can be bumped to 3.1.5 now.

**Review by martincostello — APPROVED** [review]

File: Directory.Packages.props [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <PackageVersion Include="AngleSharp" Version="0.13.0" />
+    <PackageVersion Include="JetBrains.Annotations" Version="2020.1.0" />
+    <PackageVersion Include="JustEat.HttpClientInterception" Version="3.0.0" />
+    <PackageVersion Include="MartinCostello.Logging.XUnit" Version="0.1.0" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="3.1.5" />
+    <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="3.1.5" />
+    <PackageVersion Include="Microsoft.IdentityModel.Protocols" Version="5.6.0" />
+    <PackageVersion Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageVersion Include="Shouldly" Version="3.0.2" />
+    <PackageVersion Include="System.Text.Json" Version="4.7.1" />
```
Comment (martincostello) [inline Directory.Packages.props]: Looks like 3.1.5 needs 4.7.2 for this.

**Comment by kevinchalet** [comment]
Hopefully, it should be green now... :trollface:

**Review by martincostello — APPROVED** [review]

**Comment by kevinchalet** [comment]
@martincostello thanks for the review! 👨‍🚀

**Pull request merged**